lib/kernel-manager.js

```diff
@@ -215,11 +243,11 @@ class KernelManager {
     );
   }
 
-  mergeKernelSpecs(kernelSpecs) {
+  mergeKernelSpecs(kernelSpecs: Kernelspec) {
     _.assign(this._kernelSpecs, kernelSpecs);
   }
 
-  updateKernelSpecs(callback) {
+  updateKernelSpecs(callback: Function) {
```

Comment:
Hey man, I made your recommended changes and managed to get the number of errors in kernel-manager down to 1! However, I've made some things nullable in other files that lead to some more flow errors